### DISCUSSION TITLE
Auto Wrap Data Grid Table some Resources CSV

### DISCUSSION THREAD

**Discussion by denis-dst** [GetDKAN/dkan#3997] [General]
I have some Dataset with resource in CSV
in grid view, the data table is not auto wrap
Can i setting auto wrap in code? how?
![image](https://github.com/GetDKAN/dkan/assets/61446360/b628d894-d67f-4f3b-a145-99ddcf7a1b0c)
i want data grid like this:
![image](https://github.com/GetDKAN/dkan/assets/61446360/1412b1d1-6a2d-42fd-a972-2fe23bd6ce94)

**Comment by janette** [comment]
@denis-dst sorry you may need to dig for some old recline.js documentation or try to upgrade to the newer portal.js. DKAN is now using https://github.com/GetDKAN/cmsds-open-data-components if you upgrade to DKAN 2.x
